hilti/runtime/include/types/bytes.h

```diff
@@ -487,7 +487,7 @@ class Bytes : protected std::string {
      */
     Tuple<Bytes, Bytes> split1() const {
         auto p = hilti::rt::split1(str());
-        return {p.first, p.second};
+        return tuple::make<Bytes, Bytes>(p.first, p.second);
```

Comment:
Ups, we should have moved here in the first place. I'd also opt for dropping the explicit types for maintainability.

```suggestion
        return tuple::make(std::move(p.first), std::move(p.second));
```